test/integration/scheduled-action-integration.ts

```diff
@@ -248,25 +232,20 @@ describe('Scheduled actions api', async function () {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
       const { sys, ...payload } = await scheduledAction.update()
 
-      const actions = await testSpace.getScheduledActions({
-        'environment.sys.id': payload.environment.sys.id,
-        'sys.status': 'scheduled',
-        'sys.id[in]': scheduledAction.sys.id,
+      const updatedAction = await testSpace.getScheduledAction({
+        environmentId: payload.environment.sys.id,
+        scheduledActionId: scheduledAction.sys.id,
       })
-      const updatedAction = actions.items[0]
 
       expect(updatedAction.entity).to.deep.equal(makeLink('Asset', asset.sys.id))
       expect(updatedAction.action).to.eql('unpublish')
       expect(updatedAction.scheduledFor).to.deep.equal({
         datetime: updatedSchedule,
         timezone: 'Europe/Kiev',
       })
-
-      // cleanup
-      await updatedAction.delete()
     })
 
-    test('fails to update unsupported property of scheduled actions', async () => {
+    test('fails to update with unsupported property changes in the scheduled action', async () => {
```

Comment:
This test feels like it's going to bite us in the butt someday and tests the API more than it does the client

Reply:
maybe I can just test a scheduled in the past (to test a failed update), as we don't need to rely in the specific fields 🤔

Reply:
I'd just drop it tbh. We test it in AIS

Reply:
👍🏽  done